static/js/components/UserListFormDialog.js

```diff
@@ -98,7 +98,7 @@ export default function UserListFormDialog(props: Props) {
                   value={LR_TYPE_USERLIST}
                 />
                 <label htmlFor="type-list">
-                  <span className="header">Custom List</span>
+                  <span className="header">Learning List</span>
                   Create a custom list of any of our learning resources
```

Comment:
Should `custom list` change here too?

Reply:
@pdpinch what do you think? this is in the explanation text for the option (see the screenshot above for context)

Reply:
@pdpinch bump :)

Reply:
I think we discussed just deleting the word "custom" from the explanation here, going to go with that.